bson/src/main/scala/org/mongodb/scala/bson/BsonTransformer.scala

```diff
@@ -68,7 +68,7 @@ object BsonTransformer extends DefaultBsonTransformers {}
 /**
  * Default BsonTransformers for native types.
  */
-trait DefaultBsonTransformers {
+trait DefaultBsonTransformers extends LowPrio {
```

Comment:
Had to give different priorities to `transformOption` and the collections BsonTransformer because otherwise I was having implicits errors in scala 2.13